src/Utilities/Blas.cpp

```diff
@@ -0,0 +1,18 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Utilities/Blas.hpp"
+
+namespace {
```

Comment:
I don't think putting an `extern "C"` declaration in an anonymous namespace does anything, and it seems misleading because an anonymous namespace indicates that the contents are local to the file.

Reply:
removed the anonymous namespace